mailbox/store/src/main/java/org/apache/james/mailbox/store/search/MessageSearchIndex.java

```diff
@@ -36,7 +36,14 @@
  * @param <Id>
  */
 public interface MessageSearchIndex {
-    
+
+
+    enum MessageSerachIndexCapabilities {
+        Text
```

Comment:
Not trivial to guess what means "text" in a searchIndex context. Please rename it or comment it.